src/components/DocumentModals/ShowDocumentsModal.jsx

```diff
@@ -0,0 +1,134 @@
+// React Imports
+import React, { useState } from 'react';
+// Inrupt Imports
+import { useSession } from '@inrupt/solid-ui-react';
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import ClearIcon from '@mui/icons-material/Clear';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogTitle from '@mui/material/DialogTitle';
+import Link from '@mui/material/Link';
+import Typography from '@mui/material/Typography';
+// Other Library Imports
+import { v4 as uuidv4 } from 'uuid';
+// Utility Imports
+import { getBlobFromSolid } from '../../utils';
+import ShowDocumentLocalModal from './ShowDocumentLocalModal';
+
+/**
+ * ShowDocumentsModal Component - Popup modal that shows client documents if gotten
+ * explicit permission via ACL
+ *
+ * @memberof DocumentModals
+ * @name ShowDocumentsModal
+ */
+
+const ShowDocumentsModal = ({ showModal, setShowModal, fileSrc }) => {
```

Comment:
What's the difference between this and `ShowDocumentsLocalModal`?

Reply:
Element wise, they should be the same but use different handler functions to show files. Think I would refactor the component out.

Reply:
Actually, just checked again. I mistaken this with the handleShowFile function.

The main difference between `ShowDocumentsLocalModal` and `ShowDocumentModal` is that `ShowDocumentLocalModal` shows the document in question as a modal from the `ShowDocumentsModal`. Think I'll rename `ShowDocumentLocalModal` to `ShowDocumentPreviewModal`.

Reply:
Simplified Modals into one modal